components/body/level/Level.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2006-present TrueChoice IP Holding Company, Inc.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of TrueChoice IP Holding Company, Inc.
+ * ("Confidential Information").  You shall not disclose such Confidential Information and shall
+ * use it only in accordance with the terms of the license agreement you entered into with the company.
+ */
+
+// dependencies
+import { useState } from 'react';
+// local files
+import styles from './Level.module.css';
+import LevelInfo from './LevelInfo';
+import Slider from '../../shared/Slider';
+
+/**
+ * @description - returns icon, tooltip, text, and range slider
+ * @param {object} props.levelData - level data containing levelId, name, tooltipText, icon name
+ * @return {jsx} - the Level component to render
+ */
+const Level = ({ levelData }) => {
+  // state
+  const [inputValue, setInputValue] = useState('5');
```

Comment:
lines 24-33 (e.g. `inputValue` state variable and its `onChange` handler) should be moved to `Slider.js` to encapsulate functionality in the individual atom. We don't need access to these in `Level.js` since these values are only needed within the `Slider` component.

Reply:
@achang-tcs Ready for review! *Note* - created a helper function getDefaultInputValue to handle slider's starting midpoint value.